wicket/src/main/filtered-resources/features.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could we add some XML comments here about the purpose of this file ?
From the PR comments I understand it is Apache Karaf thingy, but it would be nice to have some documentation, maybe with a link to Karaf docs about features.